sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -487,7 +487,15 @@ case class In(value: Expression, list: Seq[Expression]) extends Predicate {
     }
   }
 
-  override def toString: String = s"$value IN ${list.mkString("(", ",", ")")}"
+  override def simpleString(maxFields: Int): String = {
+    if (list.length <= maxFields) {
+      s"$value IN ${list.mkString("(", ",", ")")}"
```

Comment:
```suggestion
      s"$value IN ${list.mkString("(", ", ", ")")}"
```

Reply:
If we consider the plan output stable API (which I would do) then adding a comma here changes that and in contrast to my truncation change there is no flag to disable it. If we want to change the output format, which is nice but less valuable because not performance relevant, we should really do this separately

Reply:
> If we consider the plan output stable API

We don't

Reply:
It's not stable API, but I don't see why we should update it in an unrelated PR. This PR is for truncation, shall we open a new PR to add the space?

Reply:
@olaky How many places do you have to update? If just a couple in `explain.sql.out`, it is ok I believe.

Reply:
It's likely quite some golden files. Let's do it in followup to not block this PR.